Files/MultilingualResources/Files.ka.xlf

```diff
@@ -720,31 +720,31 @@
         </trans-unit>
         <trans-unit id="ItemSizeBytes" translate="yes" xml:space="preserve">
           <source>bytes</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">ბაიტი</target>
+          <target state="new">bytes</target>
```

Comment:
```suggestion
          <target state="translated">bytes</target>
```

Can you update the state for all the translated strings to translated?

Reply:
but how?